lib/utils.js

```diff
@@ -0,0 +1,25 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
+// Node module: loopback-boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+var fs = require('fs');
+
+exports.fileExists = fileExists;
```

Comment:
Please call the method `fileExistsSync` to follow the naming convention used by Node core for sync methods.